src/main/java/hudson/model/RadiatorView.java

```diff
@@ -52,27 +51,27 @@
 	 * User configuration - show stable builds when there are some unstable
 	 * builds.
 	 */
-	 Boolean showStable = false;
+	 Boolean showStable = Boolean.FALSE;
```

Comment:
Well, I just feel this is a bit more verbose in that case. Anyway :).

Back to the very goal of that PR itself, please revert those changes as they're unrelated to the newly introduced feature, so that the code change only contains the part related to the addition.

If you really wish to replace those, please file another dedicated PR.

Thanks!